### ISSUE TITLE
`underset` not recognized

### ISSUE THREAD

**Issue by ronaldtse** [asciidoctor/asciimath#15]
This AsciiMath expression:
```adoc
underset(_)(hat A) = hat A exp j vartheta_0
```

is being rendered as this MathML fragment:
```xml
<math>
  <mi>u</mi>
  <mi>n</mi>
  <mi>d</mi>
  <mi>e</mi>
  <mi>r</mi>
  <mi>s</mi>
  <mi>e</mi>
  <mi>t</mi>
  <mfenced open="(" close=")"/>
  <mfenced open="(" close=")">
    <mover>
      <mi>A</mi>
      <mo>^</mo>
    </mover>
  </mfenced>
  <mo>=</mo>
  <mover>
    <mi>A</mi>
    <mo>^</mo>
  </mover>
  <mi>exp</mi>
  <mi>j</mi>
  <msub>
    <mi>&#x3D1;</mi>
    <mn>0</mn>
  </msub>
</math>
```

Which gets rendered like this:
<img width="308" alt="Screen Shot 2020-04-13 at 10 24 11 PM" src="https://user-images.githubusercontent.com/11865/79128082-80c32480-7dd5-11ea-97f8-9414f8b6aed0.png">

The correct rendering (as done by MathJax) is:
<img width="159" alt="Screen Shot 2020-04-13 at 10 24 39 PM" src="https://user-images.githubusercontent.com/11865/79128117-920c3100-7dd5-11ea-930a-0b8bb9458ff9.png">

**Comment by pepijnve** [comment]
This was fixed as part of a rework of the parser in 37a11540b2d9fc5155866a135d5ae46bcb3266cb. Should be available in the next release.

**Comment by ronaldtse** [comment]
Thank you @pepijnve ! Really appreciate it! 👍
